packages/engine/README.md

```diff
@@ -210,14 +210,21 @@ $ export RUST_LOG=debug
 > **WIP** - This section is a work-in-progress. More in-depth documentation is in the works for describing all output formats and options. As such some functionality may not be mentioned here, and some functionality alluded to here might not be complete at present. 
 Currently the engine has two main form of outputs, one coming from the [json_state package](./src/simulation/package/output/packages/json_state) and the other from the [analysis package](./src/simulation/package/output/packages/analysis).
 
-#### JSON-State
-By default, the engine outputs a serialized snapshot of Agent state every step. This is written to the `./parts` folder. At the present it's possible that these files will not be valid JSON as the resultant blob may be split across multiple files (hence `part`) for buffering purposes. Development is planned for a different output format that ensures a valid JSON blob in a single file.
+At the end of each simulation run, various outputs appear within the `./<OUTPUT FOLDER>/<EXPERIMENT ID>/<SIMULATION ID>` directories. (At the moment `<OUTPUT FOLDER>` is always `./output`)
```

Comment:
Sadly, nothing appears when running a simulation. I also can't find a log entry that indicates, that an output directory is created.

Reply:
``` TRACE hash_engine::output::local::sim                                             > Making new output directory directory: "./output/3551d8c1-1e55-4c7a-bb8f-192fe6c54037/1"```

It definitely seems to be working. Lets check offline and see if you're running a valid simulation

Reply:
Sorted it, we're all good